ttnn/cpp/ttnn/operations/moreh/moreh_dot/device/moreh_dot_program_factory.cpp

```diff
@@ -7,7 +7,6 @@
 #include "tt_metal/common/constants.hpp"
 #include "tt_metal/tt_metal.cpp"
 
-
 namespace ttnn::operations::moreh::moreh_dot_op {
 MorehDotOperation::SingleCore::cached_program_t MorehDotOperation::SingleCore::create(
     const operation_attributes_t& operation_attributes,
```

Comment:
use `tensor_return_value_t& output`
No need of redeclaration of `output_tensor` later.
Use `output`.